### PR TITLE
Various updates to various main ping derived datasets

### PR DESCRIPTION
**Pull request by wlach**
The motivating factor for this were some links going out of date
due to some main summary dag changes, but a bunch of other things with
respect to the affected datasets turned out to be incorrect as well.

### CODE REVIEW

File: src/datasets/batch_view/addons/reference.md [hunk 1]
```diff
@@ -1,64 +1,27 @@
...
- |-- update_day: integer (nullable = true)
- |-- signed_state: integer (nullable = true)
- |-- is_system: boolean (nullable = true)
- |-- submission_date_s3: string (nullable = true)
- |-- sample_id: string (nullable = true)
-```
-For more detail on where these fields come from in the
-[raw data](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/environment.html#addons),
-please look
-[in the `AddonsView` code](https://github.com/mozilla/telemetry-batch-view/blob/master/GRAVEYARD.md#main-summary-clients-daily-and-addons).
```
Comment (wlach) [inline src/datasets/batch_view/addons/reference.md:62]: The schema definitions were constantly going out of date and probably not that useful to be honest. Maybe we could do better with autogenerated documentation.

File: src/datasets/obsolete/heavy_users/reference.md [hunk 2]
```diff
@@ -36,13 +36,6 @@ Example queries:
 - [Join `heavy_users` with `main_summary` to get distribution of `max_concurrent_tab_count` for heavy vs. non-heavy users](https://sql.telemetry.mozilla.org/queries/47041/source#127382)
 - [Join `heavy_users` with `longitudinal` to get crash rates for heavy vs. non-heavy users](https://sql.telemetry.mozilla.org/queries/47044/source#127385)
 
-## Scheduling
-
-This dataset is updated daily via the [telemetry-airflow](https://github.com/mozilla/telemetry-airflow) infrastructure.
-The job DAG runs every day after `main_summary` is complete.
-You can find the job definition
-[here](https://github.com/mozilla/telemetry-airflow/blob/master/dags/main_summary.py#L187-L195).
-
```
Comment (wlach) [inline src/datasets/obsolete/heavy_users/reference.md:45]: This hasn't been scheduled in a long long time

File: src/datasets/batch_view/addons/reference.md [hunk 3]
```diff
@@ -1,64 +1,27 @@
...
-
-## Example Queries
-
-## Sampling
-
-It contains one or more records for every
-[Main Summary](../main_summary/reference.md)
-record that contains a non-null value for `client_id`.
+This dataset contains one or more records for every entry in the main ping table
+that contain a non-null value for `client_id`.
```
Comment (scholtzan) [inline src/datasets/batch_view/addons/reference.md:11]:
```suggestion
that contains a non-null value for `client_id`.
```

File: src/datasets/batch_view/addons/reference.md [hunk 4]
```diff
@@ -1,64 +1,27 @@
...
-Like the Main Summary dataset, No attempt is made to de-duplicate submissions by `documentId`, so any analysis that could be affected by duplicate records should take care to remove duplicates using the `documentId` field.
-
 ## Scheduling
 
-This dataset is updated daily via the
-[telemetry-airflow](https://github.com/mozilla/telemetry-airflow) infrastructure.
+This dataset is updated daily via the [telemetry-airflow](https://github.com/mozilla/telemetry-airflow) infrastructure.
 The job DAG runs every day after the Main Summary data has been generated.
-The DAG is [here](https://github.com/mozilla/telemetry-airflow/blob/master/dags/main_summary.py#L36).
+The DAG defined in [`dags/bqetl_addons.py`](https://github.com/mozilla/telemetry-airflow/blob/master/dags/bqetl_addons.py)
```
Comment (scholtzan) [inline src/datasets/batch_view/addons/reference.md:21]:
```suggestion
The DAG is defined in [`dags/bqetl_addons.py`](https://github.com/mozilla/telemetry-airflow/blob/master/dags/bqetl_addons.py)
```

File: src/datasets/batch_view/addons/reference.md (hunk 1 above)
Comment (scholtzan) [reply src/datasets/batch_view/addons/reference.md:62]: Or maybe we can add a link to the dataset in the [schema-dictionary](https://mozilla-schema-dictionary.netlify.app/) in the future.

File: src/datasets/batch_view/events/reference.md [hunk 5]
```diff
@@ -1,66 +1,27 @@
...
-[`main_summary`](../main_summary/reference.md) is updated.
+The events dataset is updated daily.
 The job is scheduled on [Airflow](https://github.com/mozilla/telemetry-airflow).
-The DAG is [here](https://github.com/mozilla/telemetry-airflow/blob/master/dags/main_summary.py#L63).
+The DAG is defined in [`dags/copy_deduplicate.py`](https://github.com/mozilla/telemetry-airflow/blob/master/dags/copy_deduplicate.py).
+
+## Code Reference
 
-## Firefox events
+This dataset is generated by [BigQuery ETL](https://github.com/mozilla/bigquery-etl/). The query that generates the dataset is [`sql/telemetry_derived/addons_v2/query.sql`](https://github.com/mozilla/bigquery-etl/blob/master/sql/telemetry_derived/addons_v2/query.sql).
```
Comment (scholtzan) [inline src/datasets/batch_view/events/reference.md:20]:
```suggestion
This dataset is generated by [BigQuery ETL](https://github.com/mozilla/bigquery-etl/). The query that generates the dataset is [`sql/telemetry_derived/event_events_v1/query.sql`](https://github.com/mozilla/bigquery-etl/blob/master/sql/telemetry_derived/event_events_v1/query.sql).
```

File: src/datasets/batch_view/addons/reference.md (hunk 1 above)
Comment (wlach) [reply src/datasets/batch_view/addons/reference.md:62]: Yup, or (thinking slightly more near-term) if we added docs on derived datasets to bigquery-etl, an autogenerated schema definition would be high on my list of things to put in.

File: src/datasets/batch_view/addons/reference.md (hunk 3 above)
Comment (wlach) [reply src/datasets/batch_view/addons/reference.md:11]: I think `contain` is actually the correct grammar here (yes, this is confusing): https://ell.stackexchange.com/a/72639

**Pull request merged**